monitor/linuxmonitor/linuxMonitor.go

```diff
@@ -18,25 +18,48 @@ import (
 	"github.com/shirou/gopsutil/process"
 )
 
-// SystemdRPCMetadataExtractor is a systemd based metadata extractor
-func SystemdRPCMetadataExtractor(event *rpcmonitor.EventInfo) (*policy.PURuntime, error) {
+// DefaultHostMetadataExtractor is a host specific metadata extractor
+func DefaultHostMetadataExtractor(event *rpcmonitor.EventInfo) (*policy.PURuntime, error) {
 
-	if event.Name == "" {
-		return nil, fmt.Errorf("EventInfo PU Name is empty")
+	runtimeTags := policy.NewTagStore()
+
+	for _, tag := range event.Tags {
+		parts := strings.Split(tag, "=")
```

Comment:
safer to use `strings.SplitN(tag, "=", 2)`

Reply:
OK.